gems/sorbet-runtime/test/types/method_patches.rb

```diff
@@ -5,7 +5,12 @@ module Opus::Types::Test
   class MethodPatchesTest < Critic::Unit::UnitTest
     module MethodRefinement
       refine Method do
-        prepend(T::CompatibilityPatches::MethodExtensions)
+        T::CompatibilityPatches::MethodExtensions.each do |method|
```

Comment:
I am not sure what's happened here. I think you mean to iterate over the instance methods of the module, right?
```suggestion
        T::CompatibilityPatches::MethodExtensions.instance_methods(false).each do |method|
```